Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -199,6 +234,36 @@ click: function () { closeClicked.raiseEvent(this); }');
 
         return destroyObject(this);
     };
+    /**
+     * Receives a string and enters it to the DOM
+     * If the string contains a <script> tag, it is
+     * parsed using contextualFragment
+     *
+     * @type {Function}.
+     * @private
+     */
+    /**
+     *
+     * @param s {String}
+     * @param frame {Element}
+     * @param allowScripts {Boolean}
+     */
+    InfoBox.stringToHtml = function stringToHtml(s, frame, allowScripts) {
+        // if we don't have a script tag - in case the method was used as static...
+        if (allowScripts && s.indexOf('<script') > -1 && s.indexOf('</script>') > -1) {
```

Comment:
Are two code paths actually needed here?  If the sandbox is configured to not allow scripts it shouldn't matter if we assign innerHTML or create a fragment.  Can't we just always go into this check and avoid the string search and check?

Reply:
I agree that the `<script>` check is not really relevant - either fully sanitize or just leave it for the user to make sure everything is valid.
Regarding the `allowScripts` check - 

- Using the contextual fragments without allowing scripts in `sandbox` throws an error to the console if there are scripts in the HTML code.  
- InnerHTML makes sure no scripts run. 
If you think these 2 reasons are irrelevant I can remove this `if` statement.

Reply:
> Using the contextual fragments without allowing scripts in sandbox throws an error to the console if there are scripts in the HTML code.

That's a valid and desired effect in my opinion.  It let's the developer know why his scripts aren't working or let's him know that something else is trying to inject a script into his app.

> InnerHTML makes sure no scripts run.

See my comment above, this is incorrect.  Scripts can still run with innerHTML: https://github.com/AnalyticalGraphicsInc/cesium/pull/5764/files#r136330155